src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/AppointmentDtos.kt

```diff
@@ -19,6 +20,24 @@ data class CreateBookingAppointment(
   val endTime: String
 )
 
+@ApiModel(description = "The data related to a single appointment.")
+data class AppointmentDto(
+  @ApiModelProperty(required = true, value = "The event Id associated with this appointment")
+  val id: Long,
+  @ApiModelProperty(required = true, value = "The Id of the agency where the appointment is", example = "MDI")
+  val agencyId: String,
+  @ApiModelProperty(required = true, value = "The Id of the location to be used for this appointment")
+  val locationId: Long,
+  @ApiModelProperty(required = true, value = "The code for the type of appointment this is", example = "INTERV")
+  val appointmentTypeCode: String,
+  @ApiModelProperty(required = true, value = "The NOMS Id of the offender who this appointment is for")
+  val offenderNo: String,
+  @ApiModelProperty(required = true, value = "When the appointment is scheduled to start")
+  val startTime: LocalDateTime,
+  @ApiModelProperty(required = true, value = "When the appointment is scheduled to end")
+  val endTime: LocalDateTime?
```

Comment:
Should this be required false?

Reply:
Oh.. good spot.. whichever it is, it needs to be consistent with the Kotlin definition. I'll take a look and fix